glow/rate_limiter_test.go

```diff
@@ -0,0 +1,84 @@
+package glow
+
+import (
+	"testing"
+	"time"
+)
```

Comment:
It's probably good to have at least one test which checks the actual production values that will be used by the ratelimiter.